### DISCUSSION TITLE
kube-bench run on Alpine Linux?

### DISCUSSION THREAD

**Discussion by JoesSixpack** [aquasecurity/kube-bench#1474] [Questions and Help]
I tried to install kube-bench on Alpine Linux v3.18 using the alternative installation as described by [download-and-install-binaries](https://github.com/aquasecurity/kube-bench/blob/main/docs/installation.md#download-and-install-binaries) and failed.

**Comment by Mohameddgomaa** [comment]
I'm experiencing the same issue and the error message I received was: "/bin/sh: kube-bench: not found". Any assistance would be greatly appreciated.
